insights/redirect_helpers.py

```diff
@@ -0,0 +1,7 @@
+def funders(request):
+    """Rewrites query param funders to fundingOrganization for legacy 2024"""
+    if request.args.get("funders"):
+        new_query = request.query_string.decode("utf-8").replace(
```

Comment:
Howabout instead of string replace, go via `MultiDict`[^1] (it looks like this is the type of `request.args`[^2]) and update the key. I guess something like:

```python
newargs = request.args.copy()
funders = newargs.poplist("funders")
newargs.setlist("fundingOrganization", funders)

new_url = flask.url_for("data", **newargs.to_dict(flat=False)) # Using flat=False to preserve multi keys https://github.com/pallets/flask/issues/4893#issuecomment-1335957018
```
[^1]: https://werkzeug.palletsprojects.com/en/2.3.x/datastructures/#werkzeug.datastructures.MultiDict
[^2]: https://flask.palletsprojects.com/en/2.3.x/api/#flask.Request.args